app/javascript/chat/__tests__/chat.test.jsx

```diff
@@ -4,7 +4,6 @@ import { shallow } from 'preact-render-spy';
 import fetch from 'jest-fetch-mock';
 import { JSDOM } from 'jsdom';
 import Chat from '../chat';
-import algoliasearch from '../__mocks__/algoliasearchUsers';
```

Comment:
do we intend on deleting `app/javascript/chat/__mocks__/algoliasearchUsers.js` as well? i'm assuming that's coming in another PR since `app/javascript/chat/__tests__/channelDetails.test.jsx` appears to still use `algoliasearch`.

Reply:
yeah once users is moved over